docs/configuration/app.md

```diff
@@ -193,14 +193,6 @@ Dispatch [calculates](https://github.com/Netflix/dispatch/blob/develop/src/dispa
 
 ### Incident Resource Configuration
 
-#### `INCIDENT_FAQ_DOCUMENT_ID`
-
-> Controls which document id to use as the FAQ.
-
-#### `INCIDENT_CONVERSATION_COMMANDS_REFERENCE_DOCUMENT_ID`
-
-> Controls which document id to use for the conversation commands reference document.
-
```

Comment:
Shouldn't we note that they're optional rather than removing them? Otherwise, how are users going to know they're at their disposal?

Reply:
They have been removed entirely and are not used in the codebase. They will be defined by their resource type in the document table.